### ISSUE TITLE
Unsupported URL error with a specific free GDC Vault video

### ISSUE THREAD

**Issue by 3id0** [rg3/youtube-dl#5236]
Hello, I'm getting an "Unsupported URL" error with a free GDC Vault video. I've tried downloading other GDC videos and I only have the issue with this one. The error happens with both URL types : 'http://gdcvault.com/play/1020791/' and 'http://evt.dispeak.com/ubm/gdc/sf14/player.html?xml=201403828316_AJKF.xml&token=3c6c000ab0766078310c'.

Verbose output :
```
[debug] System config: []
[debug] User config: []
[debug] Command-line args: ['-v', 'http://gdcvault.com/play/1020791/']
[debug] Encodings: locale UTF-8, fs utf-8, out UTF-8, pref UTF-8
[debug] youtube-dl version 2015.03.15
[debug] Python version 2.7.2 - Darwin-12.5.0-x86_64-i386-64bit
[debug] exe versions: rtmpdump 2.4
[debug] Proxy map: {}
[generic] 1020791: Requesting header
WARNING: Falling back on generic information extractor.
[generic] 1020791: Downloading webpage
[generic] 1020791: Extracting information
ERROR: Unsupported URL: http://gdcvault.com/play/1020791/
Traceback (most recent call last):
  File "/usr/local/bin/youtube-dl/youtube_dl/extractor/generic.py", line 814, in _real_extract
    doc = parse_xml(webpage)
  File "/usr/local/bin/youtube-dl/youtube_dl/utils.py", line 1521, in parse_xml
    tree = xml.etree.ElementTree.XML(s.encode('utf-8'), **kwargs)
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/xml/etree/ElementTree.py", line 1282, in XML
    parser.feed(text)
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/xml/etree/ElementTree.py", line 1624, in feed
    self._raiseerror(v)
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/xml/etree/ElementTree.py", line 1488, in _raiseerror
    raise err
ParseError: not well-formed (invalid token): line 52, column 22
Traceback (most recent call last):
  File "/usr/local/bin/youtube-dl/youtube_dl/YoutubeDL.py", line 654, in extract_info
    ie_result = ie.extract(url)
  File "/usr/local/bin/youtube-dl/youtube_dl/extractor/common.py", line 275, in extract
    return self._real_extract(url)
  File "/usr/local/bin/youtube-dl/youtube_dl/extractor/generic.py", line 1283, in _real_extract
    raise UnsupportedError(url)
UnsupportedError: Unsupported URL: http://gdcvault.com/play/1020791/
```

**Comment by yan12125** [comment]
Thanks for the report. Such URLs will be supported by youtube-dl in the next version.

**Comment by dstftw** [comment]
This has been fixed in youtube-dl 2015.04.26. See our [FAQ](https://yt-dl.org/update) if you have problems updating.
